### ISSUE TITLE
Instructions unclear my 🍆 got stuck while slamming (1/5 ⭐️)

### ISSUE THREAD

**Issue by K4L1Ma** [veggiedefender/open-and-shut#7]
![img_20200507_144726-min](https://user-images.githubusercontent.com/6510147/81296384-174fd200-9072-11ea-997b-a4f1283a12a3.jpg)

**Comment by TheSecureTux** [comment]
This happened to me too. Please ASAP and do the needful.

**Comment by ar1ja** [comment]
how did you fix it, still having this issue 4 years later

**Comment by ahmadko** [comment]
Any updates on this one? I'm getting similar error: `Bug was detected` when calling `slammer.run()`

<img width="1574" alt="image" src="https://github.com/user-attachments/assets/c0333899-c011-4de0-bc16-a78c697e4a30">

**Comment by 4yub1k** [comment]
Yes release Patch for this asap, My friend wasn't able to solve it since first release. Now he takes his laptop every where.

**Comment by rayne-x** [comment]
Unfortunately, my laptop is totally broken

![](https://github.com/user-attachments/assets/4107f7ed-f5b9-4236-b0dd-b7446925000b)

**Comment by ar1ja** [comment]
> Unfortunately, my laptop is totally broken
> 
> ![](https://private-user-images.githubusercontent.com/32009575/367585092-4107f7ed-f5b9-4236-b0dd-b7446925000b.JPG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY0MTAwOTQsIm5iZiI6MTcyNjQwOTc5NCwicGF0aCI6Ii8zMjAwOTU3NS8zNjc1ODUwOTItNDEwN2Y3ZWQtZjViOS00MjM2LWIwZGQtYjc0NDY5MjUwMDBiLkpQRz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE1VDE0MTYzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVmZTRmOTdiYTJhNjI3OGVjYTdjNzU4OWQ4NzU4NDM4ZDI3YzI0YzJjYzY4ZTAwMTUxYzIwNjQxZTY1YWQ0OWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UmUw9op62e_1wSmMhQhcD5C6u_JQY3sCeJxc-XbNiqg)

what the fuck

**Comment by Bizarrus** [comment]
> > Unfortunately, my laptop is totally broken
> > ![](https://private-user-images.githubusercontent.com/32009575/367585092-4107f7ed-f5b9-4236-b0dd-b7446925000b.JPG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY0MTAwOTQsIm5iZiI6MTcyNjQwOTc5NCwicGF0aCI6Ii8zMjAwOTU3NS8zNjc1ODUwOTItNDEwN2Y3ZWQtZjViOS00MjM2LWIwZGQtYjc0NDY5MjUwMDBiLkpQRz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE1VDE0MTYzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVmZTRmOTdiYTJhNjI3OGVjYTdjNzU4OWQ4NzU4NDM4ZDI3YzI0YzJjYzY4ZTAwMTUxYzIwNjQxZTY1YWQ0OWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UmUw9op62e_1wSmMhQhcD5C6u_JQY3sCeJxc-XbNiqg)
> 
> what the fuck

Not what the fuck, its: what a slam!
